zap/src/main/resources/resource/icon/breakTypes/sources.txt

```diff
@@ -0,0 +1,5 @@
+<div>Icons made by <a href="https://www.flaticon.com/authors/smashicons" title="Smashicons">Smashicons</a> from <a href="https://www.flaticon.com/"                 title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/"                 title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></div>
+
+<a target="_blank" href="https://icons8.com/icons/set/javascript">JavaScript icon</a> icon by <a target="_blank" href="https://icons8.com">Icons8</a>
```

Comment:
The terms also require linking from the about dialogue:
https://icons8.com/license
https://icons8.crisp.help/en/article/where-do-i-set-the-link-irwkfh/
`Desktop software — please put the link in the About dialog`

Reply:
I don't remember which are from where but I will get all again from flaticon.